server/src/index.ts

```diff
@@ -34,7 +27,7 @@ const server = new ApolloServer({
         User,
         Mutation
     },
-    context: ({req}) => {
+    context: ({req}:{req:any}) => {
```

Comment:
https://stackoverflow.com/questions/63159116/how-can-i-correctly-type-the-context-object-in-apollo-server

See accepted (second) answer to be able to use type-safe context. 

A bit of background about TS generics if it is new: https://www.typescriptlang.org/docs/handbook/2/generics.html